package.json

```diff
@@ -11,6 +11,7 @@
     "@reduxjs/toolkit": "^1.5.1",
     "lodash": "^4.17.4",
     "prop-types": "^15.7.2",
+    "react-fps": "^1.0.6",
```

Comment:
This is only supposed to be used for our internal testing, we should remove this.
Also there should not be any change in yarn.lock and package-lock.json should not exist at all.